pageserver/src/tenant.rs

```diff
@@ -4786,12 +4786,6 @@ async fn run_initdb(
         .args(["--username", &conf.superuser])
         .args(["--encoding", "utf8"])
         .args(["--locale", &conf.locale])
-        .args(["--lc-collate", &conf.locale])
```

Comment:
Presume we’re remove the ability to pass options for this? That hard-coding is preferred?

Reply:
Sorry, passing `--locale` also sets these options. This was feedback in the previous PR that somehow didn't make it in even though I had committed the suggestion.